scripts/run_goreleaser.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+[[ -z ${DEBUG:-} ]] || set -o xtrace
+
+OS="$(uname -s | tr '[:upper:]' '[:lower:]')"
+DIR="$(cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")" && cd .. && pwd -P)"
+GITCONFIG_VOLUME=${GIT_CONFIG:-"${HOME}/.gitconfig"}
+GCLOUD_CONFIG_DIR="${CLOUDSDK_CONFIG:-${HOME}/.config/gcloud}"
+SSH_CONFIG_DIR_VOLUME="${HOME}/.ssh"
+SSH_AUTH_SOCK_ENV_VAR="/ssh-agent"
+SSH_AUTH_SOCK_VOLUME="${SSH_AUTH_SOCK}:/ssh-agent"
+
+if [[ "${BUILDKITE:-}" == "true" ]]; then
+    GITCONFIG_VOLUME="/var/lib/buildkite-agent/.gitconfig"
+    SSH_CONFIG_DIR_VOLUME="/var/lib/buildkite-agent/.ssh"
+fi
+
+# On Mac for Desktop we need to use these hardcoded values when forwarding SSH agent socket
+# Docker would 'magically' map whatever is set in $SSH_AUTH_SOCK for the user to the container ns
+if [[ "${OS}" == "darwin" ]]; then
+    SSH_AUTH_SOCK_VOLUME="/run/host-services/ssh-auth.sock:/run/host-services/ssh-auth.sock"
+    SSH_AUTH_SOCK_ENV_VAR="/run/host-services/ssh-auth.sock"
+fi
+
+DOCKER_OPTS=(
+    -w "${GO_RELEASER_WORKING_DIR}"
+    -e "GITHUB_TOKEN"
+    -e "GIT_REVISION"
+    -e "GIT_BRANCH"
+    -e "GIT_VERSION"
+    -e "BUILD_DATE"
+    -e "BUILD_TS_UNIX"
+    -e "INSTRUMENT_PACKAGE"
```

Comment:
still needed? i don't see this used anywhere in the repo

Reply:
Unused, was replaced with passing in the whole LDFLAGS list from the makefile.